apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/exception/Exceptions.kt

```diff
@@ -102,16 +102,12 @@ class HttpCacheMissException(message: String, cause: Exception? = null) : Apollo
  * [cause] might change to null and second exception will also be added as suppressed exception
  */
 class ApolloCompositeException(
-    private val firstException: Throwable?, private val secondException: Throwable?
+    firstException: Throwable?, secondException: Throwable?,
 ) : ApolloException(message = "multiple exceptions happened", secondException) {
 
-  val first: ApolloException
-    @Throws(RuntimeException::class)
```

Comment:
I think it's available in common code. We're using it there for an example: https://github.com/apollographql/apollo-kotlin/blob/31c873c5abbc4873c98e31ba17da7c16e90109ef/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/json/JsonReader.kt#L45

Reply:
yes, my bad 
Have checked the failing test cases it seem it is only available for jvm
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-throws/

Reply:
For 'native' it can only be set on AnnotationTarget.Function, while in 'common' it can be set on AnnotationTarget.PROPERTY_GETTER

Reply:
Changing to function instead of property to annotate with error for java-interop, will be a breaking change. so i think we can remove annotation for now?